### PR TITLE
Move TodoApp previews to compose-ui module

### PR DESCRIPTION
**Pull request by arkivanov**
The issue #908 is fixed and so previews can be moved to `compose-ui` module.

### CODE REVIEW

File: examples/todoapp/common/compose-ui/build.gradle.kts [hunk 1]
```diff
@@ -14,5 +16,11 @@ kotlin {
                 implementation(Deps.ArkIvanov.Decompose.extensionsCompose)
             }
         }
+
+        named("desktopMain") {
+            dependencies {
```
Comment (olonho) [inline examples/todoapp/common/compose-ui/build.gradle.kts:21]: Is it really needed with alpha3 and later?

Comment (arkivanov) [reply examples/todoapp/common/compose-ui/build.gradle.kts:21]: Seems like it works without the `ui-tooling` dependency. Should I remove it?

Comment (igordmn) [reply examples/todoapp/common/compose-ui/build.gradle.kts:21]: Let's remove it, as we transitively depend on it [here](https://github.com/JetBrains/compose-jb/blob/696d20c931feb848f113de1e67fe9446adef538f/examples/todoapp/buildSrc/src/main/kotlin/multiplatform-compose-setup.gradle.kts#L31).

We have to [explicitly](https://github.com/JetBrains/compose-jb/blob/696d20c931feb848f113de1e67fe9446adef538f/templates/multiplatform-template/common/build.gradle.kts#L20) add it only if we don't depend on the aggregated dependency `compose.desktop.common`

Comment (arkivanov) [reply examples/todoapp/common/compose-ui/build.gradle.kts]: Done

**Pull request merged**